exporter/containerimage/export.go

```diff
@@ -328,18 +328,7 @@ func (e *imageExporterInstance) Export(ctx context.Context, src *exporter.Source
 				}
 			}
 			if e.push {
-				if opts.RewriteTimestamp {
-					annotations := map[digest.Digest]map[string]string{}
-					addAnnotations(annotations, *desc)
-					// e.pushImage cannot be used because src ref does not point to the rewritten image
```

Comment:
What's the context behind removing this condition

Reply:
The comment seems outdated and the CI has no complaint on removing the code